src/templates/app/cashier/index.jsx

```diff
@@ -139,6 +139,30 @@ const Cashier = () => (
                 <DepositWithdrawButton is_payment_agent />
             </div>
         </div>
+
+        <div className='gr-padding-10' />
+
+        <div className='gr-padding-10 table-body invisible gr-parent' id='dp2p_info'>
+            <h3 className='gr-padding-10'>{it.L('DP2P')}</h3>
+            <div className='gr-row'>
+                <IconWithLink img_src={it.url_for('images/pages/cashier/dp2p_logo.svg')} />
+                <div className='gr-6 gr-8-m'>
+                    <span>{it.L('Make a deposit or withdrawal on your account by buying and selling fiat currencies from anywhere in the world.')}</span>
+                </div>
+                <div className='gr-2 gr-12-m'>
+                    <SeparatorLine className='gr-parent gr-hide gr-show-m gr-padding-10' invisible />
+                    <div className='gr-row gr-row-align-left gr-row-align-right-m'>
+                        <div className='gr-adapt'>
+                            <Button
+                                className='toggle button'
+                                href={'#TODO'}
```

Comment:
we can use the proper link now since the p2p implementation was merged to binary-static